easybuild/easyblocks/generic/pythonpackage.py

```diff
@@ -558,6 +560,15 @@ def install_step(self):
         if pythonpath is not None:
             env.setvar('PYTHONPATH', pythonpath, verbose=False)
 
+        for glob_pattern in self.cfg.get('fix_python_shebang_for', []):
+            paths = glob.glob(os.path.join(self.installdir, glob_pattern))
+            self.log.info("Fixing shebang to '%s' for files that match '%s': %s", PYTHON_SHEBANG, glob_pattern, paths)
+            for path in paths:
+                apply_regex_substitutions(path, [(r'^#!.*python$', PYTHON_SHEBANG)])
+
+                # remove backup that is made by apply_regex_substitutions
+                remove_file(path + '.orig.eb')
```

Comment:
Doesn't apply_regex have a flag to not create backup file?

Reply:
No, it doesn't. We could add one though...

Reply:
@akesandgren support for skipping backup added in https://github.com/easybuilders/easybuild-framework/pull/2844, and started using it in cc1464a31